### DISCUSSION TITLE
Issue with DataDog Integration

### DISCUSSION THREAD

**Discussion by KomalAgarwal1** [artilleryio/artillery#1512] [Artillery]
Hi Team,

I am trying to integrate DataDog via HTTP api with Artillery to push all the metrics.

After running the test, verified at DataDog in metrics, event and logs tab but could not see any stats relevant to artillery.

Can anyone please check my config and help me if I am missing something here?

Code:
config:
  plugins:
    publish-metrics:
      - type: datadog
        apiKey: "{{ $processEnvironment.DD_API_KEY }}"
        prefix: "artillery.publish_metrics_plugin."
        tags:
          - "testId:98765342"
          - "reporterType:datadog-api"
        event:
          title: "Plugin integration test"
          priority: low
          tags:
            - "testId:98765342"

**Comment by hassy** [comment]
hi @KomalAgarwal1 👋 

The config looks correct. Could you double check that `DD_API_KEY` is set? you can add a `log` statement to a scenario to do that:

```yaml
- log: "Datadog API key is: {{$processEnvironment.DD_API_KEY}}"
```

Possibly related #1489
